addons/account/wizard/account_tour_upload_bill.xml

```diff
@@ -39,5 +42,21 @@
                </form>
             </field>
         </record>
+
+        <record id="account_tour_upload_bill_email_no_alias" model="ir.ui.view">
+            <field name="name">account.tour.upload.bill.email.no.alias</field>
+            <field name="model">account.tour.upload.bill.email.no.alias</field>
+            <field name="arch" type="xml">
+                <form>
+                    <sheet>
+                        <p>Configure your email server first and send your bill by email.</p>
+                        <a type='action' name='%(action_open_settings)d' class="btn btn-link" role="button"><i class="oi oi-fw o_button_icon oi-arrow-right"/> Configure Email Servers</a>
+                    </sheet>
+                    <footer>
+                        <button string="Close" class="btn-secondary" special="cancel" data-hotkey="x" />
+                    </footer>
+               </form>
+            </field>
+        </record>
```

Comment:
With the previous comment all of that could be merge with the other template (i think)